command-line-local-shortener/main.go

```diff
@@ -88,6 +68,24 @@ func getReq(c *gin.Context) {
 	}
 }
 
+func postReq(c *gin.Context) {
+	longURL := c.PostForm("longURL")
+
+	if longURL == "" {
+		c.String(http.StatusBadRequest, "enter a url to shorten")
+		return
+	} else {
+		result := dataProccess(longURL)
+		if result != "" {
+			c.String(http.StatusOK, "Your new link is: http://localhost:8080/"+result)
```

Comment:
Change:
`"Your new link is: http://localhost:8080/"+result`
To
`fmt.Sprintf("Your new link is: %s/%s", SERVER_URL, result)`
And add a global variable `SERVER_URL` which is set to `http://localhost:8080`